### DISCUSSION TITLE
Forks page improvements

### DISCUSSION THREAD

**Discussion by shaedrich** [community/community#77824] [Repositories]
### Select Topic Area

Product Feedback

### Body

![grafik](https://github.com/community/community/assets/11225821/ac381f26-b60e-4840-8a51-aa8c52df946b)
When we look at the [forks view](https://github.com/github/explore/forks?include=active&page=1&period=2y&sort_by=last_updated) of [`github/explore`](https://github.com/github/explore), sorted by "recently updated", we immediately see, that we actually don't see that many useful information.
* **Pull Requests: 0**&ensp; We see the number of open pull requests in the fork and it doesn't come as a surprise that, depending on the use of the fork, the fork itself doesn't have its own PRs, since it's used for creating PRs against the base repository. So, seeing how many _open_ and _closed_ PRs the fork created in the base repository over time, might be much more useful in that case. Sure, there are still forks with their own PRs.
* **Branches: 2**&ensp;This information could be added, meaning the number of branches that have a base branch in the base repository

![grafik](https://github.com/community/community/assets/11225821/986d6a51-ef1a-416e-adb8-ab80fbbef00e)

**Comment by fenopa** [comment]
This was a needed page so I liked it a lot, I remember needing it a lot of times for years. I have suggestion for an item to be added to "Sort" filter as well. I think being able to sort by "commits ahead of" information would be really helpful to find actual useful forks.

**Reply by shaedrich** [reply]
Makes a lot of sense 👍🏻
